mlxtend/feature_selection/exhaustive_feature_selector.py

```diff
@@ -176,6 +186,45 @@ def __init__(self, estimator, min_features=1, max_features=1,
         # don't mess with this unless testing
         self._TESTING_INTERRUPT_MODE = False
 
```

Comment:
Add: 
```
if self.fixed_features is None:
   fixed_features = () # empty tuple
```
Then, remove `if fixed_features is not None:` (next line).

---

@rasbt: Do you think we should use type `list` or `tuple` for `fixed_features`? Tuples are immutable but, it may not be a good choice for the input. IMHO, it might be hard for the user to play with it.